build-tools/update

```diff
@@ -128,11 +138,10 @@ function updateIndex() {
 			indexContents = indexContents.replace(/<!-- news -->/g, newsData[1]);
 			console.log("DONE");
 
-			process.stdout.write("Writing new `index.html` file... ");
-			fs.writeFileSync('index.html', indexContents);
-			fs.writeFileSync('preactalpha.html', indexContents2);
-			console.log("DONE");
+			writeIndex(indexContents, indexContents2);
 		});
+	} else {
```

Comment:
I don't think it hurts to write `index.html` even if the repo isn't housed under a directory named `play.pokemonshowdown.com`... 

Out of scope for this change, but we should consider `build`-ing the `testclient.html` as well so that it actually can use cache busting...